docs/src/components/Example.js

```diff
@@ -69,7 +93,8 @@ render(<Example />, document.querySelector("#root"));`,
       'example.js': {
         content: `import React from "react";
 import "../node_modules/gestalt/dist/gestalt.css";
-import { ${components.join(', ')} } from "gestalt";
+// import "../node_modules/gestalt-datepicker/dist/gestalt-datepicker.css";
```

Comment:
This can be removed?

Reply:
I fixed it. I'm converting this logic to functions so they automatically add imports and dependencies based on used packages.